frontend/src/utils/propertyUtils.ts

```diff
@@ -65,6 +66,21 @@ export const formatAddress = (address?: IAddress) => {
   return values.join(' ') + (address?.postal ? ', ' + (address?.postal ?? '') : '');
 };
 
+/**
+ * Provides a formatted address as a string.
+ * @param address Address object from property.
+ * @returns Civic address string value.
+ */
+export const formatApiAddress = (address?: Api_Address) => {
+  const values = [
+    address?.streetAddress1 ?? '',
+    address?.streetAddress2 ?? '',
+    address?.streetAddress3 ?? '',
+    address?.province ?? '',
+  ];
+  return values.join(' ') + (address?.postal ? ', ' + (address?.postal ?? '') : '');
```

Comment:
some redundancy in this ternary. if address?.postal is falsey the ternary will return '' so there should be no need for (address?.postal ?? '')

Reply:
True. We can revisit this when we standardize the address model given the exact same ternary is used on the display of that one as well.